### DISCUSSION TITLE
Cron Jobs - Execution not at time specified

### DISCUSSION THREAD

**Discussion by Vivalldi** [vercel/community#2535] [Help] [answered]
### Summary

Daily cron jobs are **not** being executed at the time specified in the config. Being called over a half hour later

### Additional information

I have a cron job setup to run daily at 07:00 UTC. For 2 days in a row the endpoint receiving this job has been called at 07:33 UTC.

**Comment by amyegan** [comment, marked as answer]
Hi @Vivalldi. I believe this delay is because of [Hobby scheduling limits](https://vercel.com/docs/cron-jobs#hobby-scheduling-limits).

> On the hobby plan, we cannot assure a timely cron job invocation. For example, a cron job configured as `0 1 * * *` (every day at 1 am) will trigger anywhere between 1:00 am and 1:59 am.
